### PR TITLE
items and blocks

### PR DESCRIPTION
**Pull request by OroArmor**

### CODE REVIEW

File: mappings/net/minecraft/block/dispenser/BoatDispenserBehavior.mapping [hunk 1]
```diff
@@ -1,5 +1,9 @@
 CLASS net/minecraft/unmapped/C_hzrcnxyg net/minecraft/block/dispenser/BoatDispenserBehavior
+	FIELD f_gsnlaonb chestBoat Z
 	FIELD f_iqvprvdq boatType Lnet/minecraft/unmapped/C_mpfuowct$C_jyahrrif;
 	FIELD f_msafscgy itemDispenser Lnet/minecraft/unmapped/C_dnfsxzwl;
 	METHOD <init> (Lnet/minecraft/unmapped/C_mpfuowct$C_jyahrrif;)V
 		ARG 1 type
+	METHOD <init> (Lnet/minecraft/unmapped/C_mpfuowct$C_jyahrrif;Z)V
+		ARG 1 tyoe
```
Comment (NoComment1105) [inline mappings/net/minecraft/block/dispenser/BoatDispenserBehavior.mapping]:
```suggestion
		ARG 1 type
```

File: mappings/net/minecraft/item/CompassItem.mapping [hunk 2]
```diff
@@ -3,10 +3,14 @@ CLASS net/minecraft/unmapped/C_jetfzyyi net/minecraft/item/CompassItem
 	FIELD f_kxhidhak LODESTONE_TRACKED_KEY Ljava/lang/String;
 	FIELD f_mguwsctk LOGGER Lorg/slf4j/Logger;
 	FIELD f_scgvreun LODESTONE_DIMENSION_KEY Ljava/lang/String;
+	METHOD m_gyoellwj getSpawnPosition (Lnet/minecraft/unmapped/C_cdctfzbn;)Lnet/minecraft/unmapped/C_ovcqqyqp;
+		ARG 0 world
 	METHOD m_mqyuxqrl getLodestoneDimension (Lnet/minecraft/unmapped/C_hhlwcnih;)Ljava/util/Optional;
 		ARG 0 nbt
 	METHOD m_rlbiskbo hasLodestone (Lnet/minecraft/unmapped/C_sddaxwyk;)Z
 		ARG 0 stack
+	METHOD m_ucwcymfo getLoadstonePosition (Lnet/minecraft/unmapped/C_hhlwcnih;)Lnet/minecraft/unmapped/C_ovcqqyqp;
```
Comment (NoComment1105) [inline mappings/net/minecraft/item/CompassItem.mapping]: Is it not a lodestone?

**Review by NoComment1105 — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/item/CompassItem.mapping (hunk 2 above)
Comment (OroArmor) [reply mappings/net/minecraft/item/CompassItem.mapping]: oops

**Review by triphora — APPROVED** [review]
Good once NC's comments are addressed

**Review by NoahvdAa — APPROVED** [review]
Other than nc's comments, looks good

**Review by IotaBread — APPROVED** [review]

**Review by supersaiyansubtlety — APPROVED** [review]

File: mappings/net/minecraft/item/CompassItem.mapping (hunk 2 above)
Comment (OroArmor) [reply mappings/net/minecraft/item/CompassItem.mapping]:
```suggestion
	METHOD m_ucwcymfo getLodestonePosition (Lnet/minecraft/unmapped/C_hhlwcnih;)Lnet/minecraft/unmapped/C_ovcqqyqp;
```

**Pull request merged**